
Note.annotation.Demo的测试结果 【2020-03-31 22:46:29 】 
======================================================================
func1 发生异常 异常类型为：NullPointerException
异常原因为：null
----------------------------------------------------------------------------------- 
func3 发生异常 异常类型为：ArithmeticException
异常原因为：/ by zero
----------------------------------------------------------------------------------- 
======================================================================
测试结束，共发现2个异常